mobile_codes/json/countries.json

```diff
@@ -19,23 +19,22 @@
     ["Bangladesh", "BD", "BGD", "050", "470"],
     ["Barbados", "BB", "BRB", "052", "342"],
     ["Belarus", "BY", "BLR", "112", "257"],
-    ["Belgium", "BE", "BEL", "056", "206"],
+    ["Belgium", "BE", "BEL", "056", ["206", "270"]],
     ["Belize", "BZ", "BLZ", "084", "702"],
     ["Benin", "BJ", "BEN", "204", "616"],
-    ["Bermuda", "BM", "BMU", "060", ["350", "338", "310"]],
+    ["Bermuda", "BM", "BMU", "060", ["310", "338", "350"]],
     ["Bhutan", "BT", "BTN", "064", "402"],
     ["Bolivia, Plurinational State of", "BO", "BOL", "068", "736"],
     ["Bonaire, Sint Eustatius and Saba", "BQ", "BES", "535", "362"],
     ["Bosnia and Herzegovina", "BA", "BIH", "070", "218"],
     ["Botswana", "BW", "BWA", "072", "652"],
-    ["Bouvet Island", "BV", "BVT", "074", "242"],
+    ["Bouvet Island", "BV", "BVT", "074", null],
```

Comment:
[Bouvet Island](https://en.wikipedia.org/wiki/Bouvet_Island) is a dependency of Norway, and ``242`` is the MCC for Norway. I guess there is no cell reception there...